paltax/source_models_test.py

```diff
@@ -52,12 +55,17 @@
 })
 
 
-def _prepare_catalog_weights():
-    # Generate a fixed set of catalog weights.
-    catalog_weights = jnp.array([0.2, 10.0])
-    catalog_weights_cdf = jnp.cumsum(catalog_weights) / jnp.sum(catalog_weights)
-    return catalog_weights, catalog_weights_cdf
+def catalog_weights_from_hdf5(parameter):
```

Comment:
put _ in front of these functions to clarify that they are only for internal use within this test file.